.github/workflows/audit-check.yml

```diff
@@ -20,8 +19,7 @@ jobs:
       contents: read
     steps:
       - uses: actions/checkout@v3
-      - uses: dtolnay/rust-toolchain@stable
+      - uses: actions-rs/audit-check@v1
         with:
-          components: cargo-audit
-      - name: cargo audit
-        run: cargo audit
+          token: ${{ secrets.GITHUB_TOKEN }}
+          
```

Comment:
I dont know what's happening in this file, I copy pasted the contents from the main branch but it still shows as a diff :O